mmv1/third_party/terraform/services/backupdr/data_source_backup_dr_backup_plan_association_test.go.tmpl

```diff
@@ -0,0 +1,123 @@
+package backupdr_test
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataSourceGoogleBackupDRBackupPlanAssociation_basic(t *testing.T) {
+	t.Parallel()
+	context := map[string]interface{}{
+		"random_suffix": acctest.RandString(t, 10),
+	}
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
```

Comment:
```suggestion
		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
```

Reply:
Besides this change, `provider = google-beta` is needed for each resource in the configuration.